src/amberscript/parser.h

```diff
@@ -64,6 +65,10 @@ class Parser : public amber::Parser {
   Result ParseClear();
   Result ParseExpect();
 
+  Result ParseValues(const std::string& name,
```

Comment:
Method comment please.  What is the name for?  Are we appending to *values or completely overwriting.

Reply:
done